pages/docs/concepts/asyncapi-document/variable-url.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Variables in URL
+weight: 235
+---
+
+Event-driven architecture benefits from a defined base URL and rules for URL variables. You can easily manage multiple API endpoints, handling various server configurations and environments.
```

Comment:
```suggestion
```

I'd remove this paragraph entirely because it's not true. Also, let's try to avoid using terms like API endpoints when we talk about Event-driven Architecture because they may be endpoints but not APIs as widely known.